app/grandchallenge/reader_studies/templates/reader_studies/display_set_detail.html

```diff
@@ -43,8 +44,11 @@
       >
           Edit
       </button>
-    <a href="{% url 'workstations:workstation-session-create' slug=object.reader_study.workstation.slug %}?{% workstation_query display_set=object config=reader_study.workstation_config %}" class="btn btn-primary">
-      <i class="fa fa-eye"></i> View Display Set
-    </a>
+      <button class="btn btn-primary btn-sm"
+              title="View in viewer"
+              {% workstation_session_control_data workstation=object.reader_study.workstation context_object=object display_set=object config=object.reader_study.workstation_config %}
```

Comment:
The `context_object` is the `display_set` here. Should it be the `reader_study`?

Reply:
Actually shouldn't matter as only the app label is used.